src/canopy/chat_engine/query_generator/condensed.py

```diff
@@ -0,0 +1,139 @@
+import logging
+import re
+from typing import List, Optional, cast
+
+from tenacity import retry, stop_after_attempt, retry_if_exception_type
+
+from canopy.chat_engine.models import HistoryPruningMethod
+from canopy.chat_engine.prompt_builder import PromptBuilder
+from canopy.chat_engine.query_generator import QueryGenerator, LastMessageQueryGenerator
+from canopy.llm import BaseLLM, AnyscaleLLM
+from canopy.models.data_models import Messages, Query, UserMessage
+from canopy.models.api_models import ChatResponse
+
+logger = logging.getLogger(__name__)
+
+SYSTEM_PROMPT = """
+You are an expert on formulating a search query for a search engine, to assist
+ in responding to the user's question.
```

Comment:
```suggestion
SYSTEM_PROMPT = """You are an expert on formulating a search query for a search engine, to assist
 in responding to the user's question.
```

You shouldn't drop down a line in the beginning if a `"""` block that is sent to the LLM.  
From the LLM's perspective - tour system prompt now start with a `\n` token, and it has been shown that for some LLMs that completely changes the end result... 🤷‍♂️